airflow/www/package.json

```diff
@@ -87,6 +87,7 @@
     "framer-motion": "^4",
     "jquery": ">=3.5.0",
     "jshint": "^2.13.4",
+    "lodash": "^4.17.21",
```

Comment:
Should there be a corresponding `yarn.lock` diff?

Reply:
fun fact: no
We have a resolution for the same exact version of lodash. So it is already in yarn.lock and doesnt increase the size of `node_modules`